config/curator.yml

```diff
@@ -0,0 +1,24 @@
+---
+# Remember, leave a key empty if there is no value. None will be a string, not
+# a Python "NoneType"
+client:
+  hosts: [ '127.0.0.1' ]
+  port: 9200
+  url_prefix:
+  use_ssl: False
+  certificate:
```

Comment:
Shouldn't we omit the empty values ? I believe they add them in the official repo for the people to learn the available parameters